plugins/check_apt_upgrade

```diff
@@ -122,6 +129,7 @@ eval set -- "$TEMP"
 # Parse arguments
 while test "$1" != "--"; do case "$1" in
       -h) long_usage; ;;
+      -l) LIST_PACKAGES=$( "$BIN" "--package-names" 2>&1 > /dev/null ); shift;;
```

Comment:
Isn't the `2>&1 > /dev/null` but getting rid of everything?

Reply:
This is actually needed since apt-check write the file names to stderr.